types/run_parts.pp

```diff
@@ -0,0 +1,11 @@
+type Cron::Run_parts = Hash[
```

Comment:
a new type \o/
can you at a puppet-string comment with `# @summary ...` to document this type?